my-app/src/Button.tsx

```diff
@@ -0,0 +1,13 @@
+type Props = {
+  type: 'submit' | 'button';
+  btnName: string;
+  onSave: () => void;
+};
+
+export function Button({ type, btnName, onSave }: Props) {
```

Comment:
I don't find this component to be very useful. I think it's simpler to just use `<button>` directly, instead of creating a component like this. There are a few cases where a Button component could be useful, such as if you want to give it special styling and have all your buttons use that same styling. But generally, this component isn't all that useful. (The same goes for the BtnNew component.)